main.tf

```diff
@@ -20,6 +20,7 @@ resource "argocd_project" "this" {
       namespace = var.namespace
     }
 
+    # TODO verify if this should be here
     destination {
       name      = "in-cluster"
       namespace = "kube-system"
```

Comment:
These two blocks seem to be repeated. Which one should we keep?

Reply:
Discussion with @modridi : this was a configuration that was needed and stayed from the monolithic version of the devops-stack and we can delete it.